docs/reference/announcements-release-notes/release-policy.md

```diff
@@ -137,10 +137,25 @@ vast majority of dependencies Camunda _only_ applies patch updates.
 
 However, certain dependencies used by Camunda 8 may have a shorter maintenance policy than Camunda itself. Camunda may adopt a different update policy for these dependencies, as listed below.
 
-### Spring Boot
+### Spring Boot/Framework/Security Update Policies
```

Comment:
Since this is already under a "Dependency maintenance policies" header, I suggest we use:

> Spring Boot/Framework/Security updates